.github/ISSUE_TEMPLATE/3-improvement.md

```diff
@@ -0,0 +1,22 @@
+---
+name: "\U0001F310 Propose a refactoring/improvement"
+about: Use this to propose refactorings/improvements to the existing code.
+
+---
+
+<!-- 
+Use this to record any proposed refactoring/tech debt/improvement to the
+system, in order to discuss it team-wide.
+-->
+
+## Problem
+
+<!-- Describe the problem you are trying to solve. -->
+
+## Solution/implementation plan
+
+<!-- Describe your proposed solution. -->
+
+## Why do we need this?
```

Comment:
Again, I think the problem description should provide the rationale for tackling this issue.

Reply:
b7490b0